### ISSUE TITLE
Create a small chatbot that can connect to server

### ISSUE THREAD

**Issue by Triinuj** [mochan42/ft_IRC#66]

**Comment by Triinuj** [comment]
part of bonus

**Comment by pmeising** [comment]
Need help on that? :) Have free time to work on stuff if you like. :)

**Comment by Triinuj** [comment]
Sure :) I was checking the openAI api, but got stuck on quota limitations (one token per minute is limit without paying) and if we plan to finish asap, then may-be take something else.
We have to decide, what the bot does. At the moment it creates it´s own channel. It might for example have some commands it gives a response about to sender.

**Comment by pmeising** [comment]
Yeah we can hard code it and have it be an information bot of some sorts. 
I evaluated webserv the other day and they included an API to Pokedex DB.
Alternatively I was looking into hosting our own LLM like Alpaca or others on Huggingface: https://huggingface.co/
But that would not be feasable to upload to Vogsphere... XD
Let's stick to hardcoding some of it. :)

**Comment by pmeising** [comment]
Edit: Bot needs to not use any version of poll() or epoll(). Looking into using the the User infrastructure for this.
